src/averageAges.js

```diff
@@ -37,7 +42,20 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = (withChildren)
+    ? people
+      .filter(woman => (
+        people.find(person => woman.name === person.mother)
```

Comment:
it will be better to use some method here.
you dont need element. You just need to khow that it is. (boolean value)

fix in all cases

Reply:
I'm not sure that I don't need elements